src/components/Layout/Card/Card.tsx

```diff
@@ -149,8 +149,6 @@ export const CardTable = styled.table<{
     > tr:not(${CardRowDrawer}) {
       position: relative;
       display: grid;
-      // grid-template-columns: ${({ $columns }): string => $columns || `repeat(auto-fit, minmax(3rem, 1fr))`};
-      // grid-template-columns: minmax(2rem,.4fr) minmax(7rem,16rem) minmax(4rem,11rem) minmax(5rem, 7.5rem) minmax(3rem,9rem);
       grid-template-columns: 3.2rem 1fr 1fr minmax(3rem,8rem) minmax(5rem,9rem);
```

Comment:
I'm touching this in my PR so we'll have conflicts (ok ok, I will have conflicts).

Anyway, I need custom column sizes, so I'd ask to keep the parameterized version.